Plugins/ChromeDevTools/CMakeLists.txt

```diff
@@ -1,9 +1,14 @@
 set(SOURCES 
     "Include/Babylon/Plugins/ChromeDevTools.h")
 
-set(CHROME_DEVTOOLS_SUPPORTED_ENGINES "V8")
+if (WIN32)
+    set(CHROME_DEVTOOLS_SUPPORTED_ENGINES ${CHROME_DEVTOOLS_SUPPORTED_ENGINES} "JSI")
+    if (NOT WINDOWS_STORE)
```

Comment:
Is V8 CDT not working with UWP?

Reply:
No, only because we don't have a NuGet package for V8 compiled for UWP. I think going forward we're going to want to just remove V8 CDT in favor of using v8jsi, since that's where we get the inspector code from anyway.

Reply:
Gotcha. Was the logic was missing the `WINDOWS_STORE` check before?

Reply:
It's possible. I think when I originally wrote it I was under the false assumption that `WINDOWS_STORE` and `WIN32` were mutually exclusive, and I never hit this path on UWP since V8 isn't available for UWP in the first place.

Reply:
Yeah, I didn't know this either before. `WIN32` is for all windows platforms (Win32 and UWP). `WINDOWS_STORE` is an additional thing to check to differentiate.